change/@fluentui-react-avatar-65887485-78bd-4b63-812e-75334eccc841.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Remove theme-provider dep",
```

(nit) avoid abbrvs. in release comments
```suggestion
  "comment": "Remove @fluentui/react-theme-provider dependency",
```